README.md

```diff
@@ -1,23 +1,32 @@
 ## SageMaker Code Editor
 
-This is the repo for `sagemaker-code-editor`.
+This is the repo for `sagemaker-code-editor`. For more information on Code Editor on SageMaker, please refer to: https://docs.aws.amazon.com/sagemaker/latest/dg/code-editor.html
 
 The `patched-vscode` folder's only usage is to help reviewers review the patch changes. To ease reviewing patches (`.diff` files), whenever we raise a PR for adding/updating a patch, we will also apply the patches to this directory so that this folder's diff shows up in the PR.
 
 Whenever we create a new branch with `vscode` pointing to a specific commit, this folder must be populated with the same contents as `vscode` at that particular commit.
 
+## Patching Execution
+
 To properly patch, please follow instructions below:
 
+* After cloning the repo, run `git submodule init` and `git submodule update` to initialize the `vscode` submodule.
+* Use `quilt` to apply patches in sequence using `quilt push -a` from the root directory. 
```

Comment:
seems quilt is amazon internal concept? Is this readme surfaced to customers?

Reply:
`quilt` is a public linux technology: https://linux.die.net/man/1/quilt